docs/reference/execute-methods.md

```diff
@@ -68,6 +68,21 @@ offset | number | no | The value in range [0.01, 0.5]. It defines how far from p
 value | string | no | If provided WDA will try to automatically scroll in the given direction until the actual picker value reaches the expected one or the amount of scrolling attempts is exceeded. | myvalue
 maxAttempts | number | no | The maximum number of scrolling attempts to reach `value` before an error will be thrown. Only makes sense in combination with `value`. 25 by default | 50
 
+### mobile: sendMemoryWarning
+
+Simulates sending of Low Memory warning to the target application.
+It might be useful to verify the
+[didReceiveMemoryWarning](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621409-didreceivememorywarning?language=objc)
+API in the application under test.
+This feature only works on real devices running iOS 17+ with Xcode 15+ SDK.
+The target application must be running while this API is called.
```

Comment:
btw, `devicectl` basically has overhead (1 sec - a few seconds at least on my local every run), so would it also be helpful to leave a comment about the CLI's performance in code, or leave a note here about this command depends on `devicectl`...? (i have no strong opinion, but just wondered it would be helpful to address the CLI itself's overhead, by outside Appium)

Reply:
It is probably a bug that Apple ought to fix at some time. I would not worry about this API in particular, as it is not going to be called frequently and the actual call performance is not critical there.